src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs

```diff
@@ -17,22 +17,48 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
```

Comment:
I think what Sudhakar means is that you can put "System" related usings above our internal usings. You can configure this in VS.